levant/deploy.go

```diff
@@ -505,6 +505,7 @@ func (l *levantDeployment) dynamicGroupCountUpdater() error {
 	return nil
 }
 
+// Checks that all task groups have a count bigger than zero.
```

Comment:
func comments should start with the function name

Reply:
Makes sense. I added the function name `isJobZeroCount` to the beginning of the comment. Hope that was correct.